### DISCUSSION TITLE
ModuleNotFoundError: No module named 'bottle'

### DISCUSSION THREAD

**Discussion by jscolli1** [FlareSolverr/FlareSolverr#1400] [General]
Can anyone help me with this error? I'm trying to run FlareSolverr for the first time but keep running into issues. I did a few searches but didn't see this error mentioned anywhere. Any help would be awesome! Thanks in advance!

`Traceback (most recent call last):
  File "/Users/jscolli2/FlareSolverr/src/flaresolverr.py", line 7, in <module>
    from bottle import run, response, Bottle, request, ServerAdapter
ModuleNotFoundError: No module named 'bottle'`

**Comment by ilike2burnthing** [comment]
https://github.com/FlareSolverr/FlareSolverr#from-source-code
